app/preprints/-components/submit/author-assertions/styles.scss

```diff
@@ -0,0 +1,15 @@
+// stylelint-disable max-nesting-depth, selector-max-compound-selectors
+
+
+.preprint-input-container {
```

Comment:
This style is the same as the wrapper div in submit/template.hbs (`preprint-state-machine-container`), so the wrapper div here is redundant